docs/release_new_version.md

```diff
@@ -0,0 +1,96 @@
+
+## Set current version in staging
+
+Once the discovery has finished, you can proceed to set the build as the current release version in staging.
+(The idea is to first test in staging as a last check before bouncing the version in prod)
+
+Once there, `ce --env staging builds set_current gh-xxxx` will set the current build version
+for staging to your identified build id.
+
+Note that `--env staging` comes before any command, and its default value is always `staging`,
+but uses it explicitly to be more clear as to what is running for which environment.
+
+After passing the sanity checks (it will ask you to confirm what branch this comes from and in what env you're running the command in), the build has been marked as current.
+Note that these sanity checks are present in most commands,
+and we're always on the lookout for more places to put them, so if you find anything not secured by these checks,
+please do let us know.
+
+_If this fails due to some hash missmatch errors, you need to bump the hack version number in `webpack.config.esm.js`._
+
+## Bringing staging up
+
+Now staging needs to be brought up. This is done with `ce --env staging environment start`.
+The message should be that the number of instances has been increased from 0 to 1 (Or more!)
+
+If this is not the case, it means that staging was already up and you'll need to instead refresh those instances.
+This is acomplished by running `ce --env staging environment restart`.
+This can take a bit until all the new nodes are healthy.
+
+Once this is done, the new version is running in https://godbolt.org/staging
+(You can confirm it by going to Other > Version Tree, which points to the release commit in GitHub)
+
+## Bringing staging down
+
+Unless staging is currently in use for something else, you should now always remember to bring staging down.
+
+This is unsuprisingly acomplished by running `ce --env staging environment stop`.
+
+## Mark discovery run as safe for production
+
+Instead of now rerunning the first step of compiler discovery for your commit but now for the prod environment,
+you can instead run `ce --env staging safeforprod gh-xxxx`, and it will create a compiler discovery result for prod.
+
+## Set current version in prod
+
+Now that you've tested that everything works correctly, 
+unning `ce --env prod builds set_current gh-xxxx` will mark your build as current for the prod environment.
+
+_If this fails complaining that prod is currently bounce locked, it means that someone has blocked prod from updating.
+The usual reason is that a conference is currently running and we don't want to have any big changes at this moment.
+There are instructions in the error message on how to bypass this if necessary, but ask around to check first._
+
+## Refresh prod
+
+Now that the version is set as current, running `ce --env prod environment refresh` will refresh the live instances,
+bringing them up-to-date with your new version. Go make your preferred brevage,
```

Comment:
Having reached this point I will now go make my preferred beverage! Thanks for the excellent docs, very straightforward :)

Reply:
Heheh :) nit: typo:
```suggestion
bringing them up-to-date with your new version. Go make your preferred beverage,
```